tools/gcs-upload/index.ts

```diff
@@ -2,6 +2,8 @@ import { Storage } from '@google-cloud/storage';
 import path from 'path';
 import fs from 'fs-extra';
 import { fileURLToPath } from 'url';
+import pLimit from 'p-limit';
```

Comment:
mmm, I think don't use new package there

Reply:
There are a lot of new dependencies in the pull request 😅